docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -159,6 +159,26 @@ See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getA
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
+[discrete]
+[[synthetics-create-test-script-recorder]]
+=== Synthetics Recorder
+
+beta[]
+
+// In what circumstances should you use Synthetics Recorder?
+Use the Synthetics Recorder to generate code for one or more steps by interacting with a web page directly.
+
+// Some high-level details on what it does
+The recorder launches a special https://www.chromium.org/Home/[Chromium browser] that will listen to each interaction you have with the web page and record them internally.
+When you're done interacting with the page, the recorder converts the recorded actions into usable JavaScript code.
```

Comment:
Very minor point, feel free to ignore it :) 
Should we mention that the js file is only executable by Synthetics agent?
```suggestion
When you're done interacting with the page, the recorder converts the recorded actions into usable JavaScript code that can be executed by Synthetics agent.
```

Reply:
```suggestion
When you're done interacting with the page, the recorder converts the recorded actions into JavaScript code for use with Elastic Synthetics.
```

Maybe even something like this, dropping `usable` altogether and highlighting that the code is only intended for use with Synthetics?